Zadaca2/Numbers.java

```diff
@@ -0,0 +1,128 @@
+
+import java.util.*;
```

Comment:
It's generally a good practice to import only needed package, not the complete one. For example: java.util.ArrayList instead of java.util.*